tests/unittests/Foundation/NSNotificationTests.mm

```diff
@@ -15,7 +15,54 @@
 //******************************************************************************
 
 #include <TestFramework.h>
+#import <Starboard/SmartTypes.h>
 #import <Foundation/Foundation.h>
+
+// Avoid "property access result unused" warnings by _consume-ing the value of a property accessor.
+template <typename... T>
+static inline void _consume(T&&...) {

```

Comment:
I have a sudden aversion to blanket disabling this warning, but I don't know why.